units/host_termination.go

```diff
@@ -34,8 +34,9 @@ func init() {
 }
 
 type hostTerminationJob struct {
-	HostID   string `bson:"host_id" json:"host_id" yaml:"host_id"`
-	job.Base `bson:"metadata" json:"metadata" yaml:"metadata"`
+	HostID          string `bson:"host_id" json:"host_id"`
+	TerminateIfBusy bool   `bson:"terminate_if_busy" json:"terminate_if_busy"`
+	job.Base        `bson:"metadata" json:"metadata"`
 
 	host *host.Host
```

Comment:
should we turn this into not-a-pointer to avoid needing to dereference/copy it a bunch?

Reply:
I think a pointer is the right way to go here so we aren't passing copies around. Some of the functions used here require a pointer anyway so this is slightly less messy